src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Data Resilience
+summary: An overview of how CockroachDB deployments can achieve resilience with built-in features and tooling.
+---
+
+CockroachDB provides built-in [_high availability_](#high-availability) features and [_disaster recovery_](#disaster-recovery) tooling to achieve operational resiliency in various deployment topologies and use cases. To maximize uptime, high availability features ensure continuous access to data without interruption even in the presence of failures or disruptions. Disaster recovery tools allow for recovery from major incidents to minimize downtime and data loss.
+
+<image src="{{ 'images/v24.3/resilience.svg' | relative_url }}" alt="Diagram showing how high availability features and disaster recovery tools create a resilient CockroachDB deployment." style="max-width:100%" />
+
+You can balance required SLAs and [recovery objectives]({% link {{ page.version.version }}/disaster-recovery-overview.md %}) with the cost and management of each of these features to build a resilient deployment.
+
+{{site.data.alerts.callout_success}}
+For a practical guide on how CockroachDB uses Raft to replicate, distribute, and rebalance data, refer to the [Replication and Rebalancing]({% link {{ page.version.version }}/demo-replication-and-rebalancing.md %}) demo.
```

Comment:
since you mention in another comment drawing a Raft vs cross-cluster, maybe worth adding "... uses Raft to consistently replicate ..."

Reply:
I have changed the name of the page. I have also done this for v24.2 to make the link changing make more sense.

Reply:
Note I haven't added the Data Resilience page in v24.2; mainly because we will not have LDR in v24.2 and so a Resilience page will be look different in this version. Happy to add to v24.2 as well know if we want to.